documentation/slate/source/includes/apis.md

```diff
@@ -1029,157 +1045,134 @@ Other parameters to come...
-                date_times: [{
-                    date_time: "20160616T093300",
-                    additional_informations: [],
-                    links: [{
-                        type: "vehicle_journey",
-                        value: "vehicle_journey:RAT:RATRM1REGA9828-1_dst_2",
-                        rel: "vehicle_journeys",
-                        id: "vehicle_journey:RAT:RATRM1REGA9828-1_dst_2"
+                "date_times": [{
+                    "date_time": "20160616T093300",
+                    "additional_informations": [],
+                    "links": [{
+                        "type": "vehicle_journey",
+                        "value": "vehicle_journey:RAT:RATRM1REGA9828-1_dst_2",
+                        "rel": "vehicle_journeys",
+                        "id": "vehicle_journey:RAT:RATRM1REGA9828-1_dst_2"
                     }],
-                    data_freshness: "base_schedule"
+                    "data_freshness": "base_schedule"
                 }, {
-                    date_time: "20160617T094400",
-                    additional_informations: [],
-                    links: [{
-                        type: "vehicle_journey",
-                        value: "vehicle_journey:RAT:RATRM1REGA9827-1_dst_2",
-                        rel: "vehicle_journeys",
-                        id: "vehicle_journey:RAT:RATRM1REGA9827-1_dst_2"
+                    "date_time": "20160617T094400",
+                    "additional_informations": [],
+                    "links": [{
+                        "type": "vehicle_journey",
+                        "value": "vehicle_journey:RAT:RATRM1REGA9827-1_dst_2",
+                        "rel": "vehicle_journeys",
+                        "id": "vehicle_journey:RAT:RATRM1REGA9827-1_dst_2"
                     }],
-                    data_freshness: "base_schedule"
+                    "data_freshness": "base_schedule"
                 }]
             }]
         },
-        additional_informations: null,
-        links: [],
-        geojson: {}
+        "additional_informations": null,
+        "links": [],
+        "geojson": {}
     }]
 }
 ```
 
+
+Also known as `/route_schedules` service.
+
+This endpoint gives you access to schedules of routes, with a response made
+of an array of [route_schedule](#route-schedule), and another one of [note](#note). You can
+access it via that kind of url: <https://api.navitia.io/v1/{a_path_to_a_resource}/route_schedules>
+
+### Accesses
+
+| url | Result |
+|-------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| `/coverage/{resource_path}/route_schedules`  | List of the entire route schedules for a given resource                                                       |
+| `/coverage/{lon;lat}/route_schedules`        | List of the entire route schedules for coordinates                                                       |
```

Comment:
for those url, there is the same problem for stop_schedules, places nearby, departures, arrivals.
In "Public Transportation Objects exploration" we mention regions, and not around coordinates, which is more correct